index.html

```diff
@@ -64,44 +74,53 @@ <h2 class="head2">Tonic</h2>
                         <div class="link3">javaScript</div>
                     </li>
                 </ul>
-                <button class="see-project1">
+                <button class="see-project1 unique2">
```

Comment:
As requested by the previous reviewer, please update the following `See Project` button color 😉.
Your design:
![image](https://user-images.githubusercontent.com/42799579/206871722-f1a0a4d0-361a-496a-9c46-a216a75867ce.png)

Kindly notice that, in Figma, there is a mouse hovering over this card. This is showing to you that the color of the button should be blue only after hovering, the default color should be just like the other buttons 😄.
Figma design:
![image](https://user-images.githubusercontent.com/42799579/206871703-6dad1f8a-85e7-48d4-b9f8-99719a793bc8.png)

Expected button color:
![image](https://user-images.githubusercontent.com/42799579/206871747-7c6eda69-1253-48a4-aa4a-9175030341a3.png)